packages/react-core/src/components/Nav/NavExpandable.tsx

```diff
@@ -141,7 +141,12 @@ class NavExpandable extends Component<NavExpandableProps, NavExpandableState> {
                 </button>
               )}
             </PageSidebarContext.Consumer>
-            <section className={css(styles.navSubnav)} aria-labelledby={this.id} hidden={expandedState ? null : true}>
+            <section
+              className={css(styles.navSubnav)}
+              aria-labelledby={this.id}
+              hidden={expandedState ? null : true}
+              {...(!expandedState && { inert: true })}
```

Comment:
Looks like we need to pass an empty string to apply the attribute rather than a boolean:

```suggestion
              {...(!expandedState && { inert: '' })}
```